lib/rational/include/Rational.h

```diff
@@ -1,33 +1,36 @@
 #pragma once
 
-
+#include <iostream>
```

Comment:
Replace with #include <iosfwd>
Please move #include <iostream> to cpp file

Reply:
What for?

Reply:
It's better to use forward declarations, isn't it?

Reply:
Why is it better?